src/DynamoCore/Library/FunctionDescriptor.cs

```diff
@@ -3,7 +3,7 @@
 using System.IO;
```

Comment:
Same decision here, `LibraryCustomizationServices` is kept as `static` for now, so `IPathManager` needs to be passed into them for now.